license_copyright/license.yml

```
@@ -0,0 +1,33 @@
+name: 'Update copyright license'
+description: Update license years automatically
+author: "TheAlgorithms"
+inputs:
+  filename:
+    description: License file name
+    required: true
+    default: LICENSE
+runs:
+  using: composite
+  steps:
+    - name: Update the License
+      run: |
+        (echo "Copyright (C) 2016-$(date +"%Y") TheAlgorithms and contributors";  tail -n +2 ${{ inputs.filename }}) > License.tmp
+        mv License.tmp ${{ inputs.filename }}
+    - name: Setup Git configurations
+      shell: bash
+      run: |
+        git config --global user.name github-actions[bot]
+        git config --global user.email 'github-actions@users.noreply.github.com'
+    - name: Commit to a new branch and push changes
+      shell: bash
+      run: |
+        git checkout -b license-update-${{ github.sha }}
+        git commit -m "docs: updating Copyright license year"
+        git push
+    - name: Creating and merging the PR
+      shell: bash
+      if [[ `git status --porcelain` ]]; then
+      run: |
```

Whoops, made a mistake. 😅
```suggestion
      shell: bash
      run: |
          if [[ `git status --porcelain` ]]; then
```